src/components/AmountTextInput.js

```diff
@@ -64,6 +65,7 @@ class AmountTextInput extends React.Component {
         keyboardAppearance='dark'
         keyboardType='numeric'
         placeholder={placeholder}
+        placeholderTextColor={COLORS.midContrastDetail}
```

Comment:
Is it the same color for both iOS and Android?

Reply:
Yes, all colors are now the same between operating systems.